src/sentry/static/sentry/app/views/settings/organizationTeams/index.jsx

```diff
@@ -1,103 +0,0 @@
-import PropTypes from 'prop-types';
-import React from 'react';
-import Reflux from 'reflux';
-import createReactClass from 'create-react-class';
-
-import {loadStats} from 'app/actionCreators/projects';
-import {sortArray} from 'app/utils';
-import ProjectsStore from 'app/stores/projectsStore';
-import SentryTypes from 'app/sentryTypes';
-import TeamStore from 'app/stores/teamStore';
-import withApi from 'app/utils/withApi';
-import withOrganization from 'app/utils/withOrganization';
-
-import OrganizationTeams from './organizationTeams';
-
-const OrganizationTeamsContainer = createReactClass({
-  displayName: 'OrganizationTeamsContainer',
-
-  propTypes: {
-    api: PropTypes.object,
-    organization: SentryTypes.Organization,
-    routes: PropTypes.arrayOf(PropTypes.object),
-  },
-
-  mixins: [
-    Reflux.listenTo(TeamStore, 'onTeamListChange'),
```

Comment:
Removed dependency on `TeamStore`, instead keep teams in state, use `user-teams` endpoint and `teams` endpoint with option to only include teams user is not a member of (this is needed for accurate pagination/filter). We don't actually paginate, but need to show a message when there are more available teams.